spec/14-classes.md

```diff
@@ -1913,6 +1941,34 @@ case, the `Closure` object is empty.
 
 Closure objects can not be serialized or unserialized.
 
+###Class `DivisionByZeroError`
+
+An instance of this class is thrown when an attempt is made to divide a number by zero using the remainder operators ([`%`](10-expressions.md#multiplicative-operators) and [`%=`](10-expressions.md#compound-assignment)). The class type is defined, as follows: 
```

Comment:
It can also be thrown by `intdiv`, but I guess that's outside the scope of the spec?